crates/devolutions-pedm-shared/src/policy.rs

```diff
@@ -309,23 +310,26 @@ pub struct ElevationConfigurations {
 #[repr(transparent)]
 #[derive(Serialize, Deserialize, JsonSchema, Debug, PartialEq, Eq, Hash, Clone)]
 #[serde(try_from = "String")]
-pub struct Id(String);
+pub struct Id(Uuid);
```

Comment:
suggestion: I think it’s fine to just use `Uuid` directly, and remove the `Id` type which was just a wrapper for validating the string-based ID using a Regex.

Reply:
I've done this too; I thought of it earlier but didn't bother. But actually: fearless refactoring FTW!